src/main/java/mate/academy/spring/dto/request/UserRegistrationDto.java

```diff
@@ -0,0 +1,38 @@
+package mate.academy.spring.dto.request;
+
+import javax.validation.constraints.Size;
+import mate.academy.spring.validator.Email;
+import mate.academy.spring.validator.RepeatPassword;
+
+@RepeatPassword
```

Comment:
Let's rename the annotation to be more accurate e.g.`Password` or `ValidPassword`